modules/installation-guide/partials/proc_creating-an-instance-of-the-che-operator.adoc

```diff
@@ -1,44 +1,60 @@
 = Creating an instance of the {prod} Operator
 
-Follow this procedure to install {prod} with the default configuration. To modify the configuration, see xref:configuring-the-che-installation.adoc[].
+Follow this procedure to install {prod-short} with the default configuration. To modify the configuration, see xref:configuring-the-che-installation.adoc[].
 
 .Procedure
 
-. To create an instance of the {prod} Operator, in the left panel, navigate to the *Operators* -> *Installed Operators* section.
+<<<<<<< HEAD
+. Using the left panel, navigate to the *Operators* -> *Installed Operators* page.
 
-. In the *Installed Operators* screen, click the *{prod}* name.
+. In the *Installed Operators* page, click the *{prod}* name.
 
-. In the *Operator Details* screen, in the *Details* tab, inside of the *Provided APIs* section, click the *Create Instance* link.
+. In the *Operator details* page, in the *Details* tab, click the *Create instance* link in the *Provided APIs* section.
++
+A user is navigated to the *Create CheCluster* page, which contains the configuration stored in the `CheCluster` Custom Resource, needed to create a {prod-short} instance. 
+
+. Create the *{prod-checluster}* cluster using the btn:[Create] button at the end of the page using the default values.
+
+. In the *Operator details* page, in the *{prod} Cluster* tab, click the *{prod-checluster}* link.
+=======
+. In the left panel, navigate to the *Operators* -> *Installed Operators* page.
 
-. The *Create CheCluster* page contains the configuration of the overall {prod-short} instance to create. It is the `CheCluster` Custom Resource. Keep the default values. 
+. On the *Installed Operators* page, click the *{prod}* name.
 
-. To create the *{prod-checluster}* cluster, click the btn:[Create] button in the lower left corner of the window.
+. On the *Operator details* page, in the *Details* tab, in the *Provided APIs* section, click the *Create instance* link.
 
-. On the *Operator Details* screen, in the *{prod} Cluster* tab, click on the *{prod-checluster}* link.
+. The *Create CheCluster* page contains the configuration of the {prod-short} instance to create. It is the `CheCluster` Custom Resource. Keep the default values. 
 
-. To navigate to the *{prod-checluster}* instance, click the link under *{prod} URL*. 
+. To create the *{prod-checluster}* cluster, click the btn:[Create] button at the end of the page.
+
+. On the *Operator details* page, in the *{prod} Cluster* tab, click on the *{prod-checluster}* link.
+>>>>>>> 14bce3a0 (Update Installing Che on OpenShift 4)
+
+. Navigate to the *{prod-checluster}* instance using the link displayed under the *{prod} URL* output. 
 +
 [NOTE]
 ====
-The installation might take more than 5 minutes. The URL appears after the {prod} installation finishes. 
+The installation might take more than 5 minutes. The URL appears when the {prod} installation finishes. 
 ====
 
 .Verification steps
```

Comment:
According to the new mod docs templates, this should now be just `.Verification`